.rb

```diff
@@ -0,0 +1,75 @@
+=begin
+Write Ruby code with statements that practice conditional logic,
+using if, elsif, and else when appropriate
+Write Ruby code with statements that practice loops, like while and until when appropriate
+Write Ruby code that takes in user input using gets.chomp
+Practice creating, storing, and modifying variables
+The program should ask the user for an operation (as a string symbol) and two numbers.
+=end
+
+puts "Welcome to Ruby Calculator!"
+puts "Choose one of the following operators: "
+operators = ["add", "subtract", "multiply", "divide", "+", "-", "*", "/"]
+puts operators
+
+#user inputs command
+command = gets.chomp.downcase
+
+case command
```

Comment:
Since this case has the same result no matter the operation chosen, you could do it with one when line,

```ruby
case command
  when 'add', '+', 'subtract', '-', 'multiply', '*', 'divide', '/'
     puts 'that operation is valid'
  else
    puts 'please enter a valid operation'
end
```

Further this doesn't do anything except output to the user, the loop below can do that as well.